packages/devilspie/build.sh

```diff
@@ -0,0 +1,16 @@
+TERMUX_PKG_HOMEPAGE=https://wiki.gnome.org/Projects/DevilsPie
+TERMUX_PKG_DESCRIPTION="Utility for matching X11 windows and performing actions on them"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="Rafael Kitover <rkitover@gmail.com>"
+TERMUX_PKG_VERSION=0.23
+TERMUX_PKG_SRCURL=http://deb.debian.org/debian/pool/main/d/devilspie/devilspie_${TERMUX_PKG_VERSION}.orig.tar.xz
+TERMUX_PKG_SHA256=2dd9d4b7424ac999f20c214b0ff71e8c5836d97003d7a8a7391105cd16d36deb
+TERMUX_PKG_DEPENDS="libx11, gettext"
+TERMUX_PKG_BUILD_DEPENDS="intltool"
+
+termux_step_pre_configure() {
+	patch -p1 -i ${TERMUX_PKG_BUILDER_DIR}/fix_manpage_lintian_warnings.patch
+	patch -p1 -i ${TERMUX_PKG_BUILDER_DIR}/fix_memleak_in_my_wnck_get_viewport_start.patch
+	patch -p1 -i ${TERMUX_PKG_BUILDER_DIR}/fix_using_deprecated_wnck_functions.patch
+	patch -p1 -i ${TERMUX_PKG_BUILDER_DIR}/remove_unavailable_options_from_manpage.patch	
```

Comment:
You can't apply patches in this way.

Reply:
https://github.com/termux/termux-packages/blob/master/build-package.sh#L374

https://github.com/termux/termux-packages/blob/master/scripts/build/termux_step_patch_package.sh#L1